### PR TITLE
[FIX] Migrate DocuSign to Comply Exchange

### PR DESCRIPTION
**Pull request by Locrian24**
## Description of the change

> Migrates our tax submission service from DocuSign to Comply Exchange, and all the UI implications of this change.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: (PUT IT HERE)
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature [hunk 1]
```diff
@@ -136,3 +102,19 @@ Feature: Docusign Form
...
+    And the "Complyexchange Thank you page Exit" event is fired
+    When the document fails to complete
+    Then a general error banner is shown
+    And the banner has title
+      """
+      There was a problem submitting your information
+      """
+    And the banner has description
+      """
+      Please review your information and try again. If this problem continues, contact Support.
```
Comment (Ultha258) [inline packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature:119]: @Locrian24 Is 'contact support' a link?

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature [hunk 2]
```diff
@@ -19,14 +19,15 @@ Feature: Tax And Cash Dashboard
...
 
-    Examples: 
-      | documentType |
-      | W9           |
-      | W8 BEN       |
-      | W8 BEN-E     |
+    Examples:
+      | requiredDocumentType | documentType |
+      | W9                   | W-9          |
+      | W8BEN                | W-8          |
```
Comment (Ultha258) [inline packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:29]: @Locrian24 : 'W8-BEN'

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature [hunk 3]
```diff
@@ -19,14 +19,15 @@ Feature: Tax And Cash Dashboard
...
+      | W8BENE               | W-8          |
```
Comment (Ultha258) [inline packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:30]: @Locrian24 : 'W8-BEN-E'

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature [hunk 4]
```diff
@@ -84,11 +85,11 @@ Feature: Tax And Cash Dashboard
...
-      | NOT_ACTIVE | W8-BEN-E     | W8-BEN tax form is invalid.                                                | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN-E form. |
+    Examples:
+      | status     | documentType | taxAlertHeader                                                              | taxAlertMessage                                                                                                                 |
+      | NOT_ACTIVE | W9           | Your W-9 tax form has personal information that doesn't match your profile. | Please resubmit a new W-9 form.                                                                                                 |
+      | NOT_ACTIVE | W8-BEN       | W-8 tax form is invalid.                                                    | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN form.   |
```
Comment (Ultha258) [inline packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:91]: @Locrian24 looking at the W-9 message, could be add 'Your' to the start of the sentence for consistency? This is  a nice to have, so if it requires more work (other than a quick update), let's leave it

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature [hunk 5]
```diff
@@ -84,11 +85,11 @@ Feature: Tax And Cash Dashboard
...
+      | NOT_ACTIVE | W8-BEN-E     | W-8 tax form is invalid.                                                    | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN-E form. |
```
Comment (Ultha258) [inline packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:92]: Same as above

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-form.feature [hunk 6]
```diff
@@ -115,16 +115,17 @@ Feature: Tax Form Flow
     Given a brand based in <brandCountry>
```
Comment (Ultha258) [inline packages/mint-components/src/components/tax-and-cash/sqm-tax-form.feature:115]: @Locrian24 this is for sentence above (couldn't add a comment to that one)
update docusign with 'Comply Exchange'

**Review by Ultha258 — CHANGES_REQUESTED** [review]

File: packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature (hunk 1 above)
Comment (Locrian24) [reply packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature:119]: I don't believe so

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature (hunk 2 above)
Comment (Locrian24) [reply packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:29]: `requiredDocumentType` refers to the value sent by the backend, which doesn't include dashes

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature (hunk 3 above)
Comment (Locrian24) [reply packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:30]: Same as above

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature (hunk 4 above)
Comment (Locrian24) [reply packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:91]: Just cause it's a prop on the component means it's a bit of a pain to update. Definitely doable but will need to be updated in a bunch of different places and add to the release scope a bit.

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature (hunk 2 above)
Comment (Ultha258) [reply packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:29]: I see, thank you!

File: packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature (hunk 4 above)
Comment (Ultha258) [reply packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature:91]: nvm then, we do want to wrap the project up soon, probably don't have time for the change

**Review by Ultha258 — DISMISSED** [review]

File: packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form-view.tsx [hunk 7]
```diff
@@ -263,95 +266,43 @@ export const DocusignFormView = (props: DocusignFormViewProps) => {
       )}
       {/* 
       // @ts-ignore */}
```
Comment (00salmon) [inline packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form-view.tsx:268]:
```suggestion
      {/* 
      // @ts-ignore */}
```
not sure if this is necessary

**Review by 00salmon — COMMENTED** [review]
just some minor comments, looking good otherwise

**Review by 00salmon — APPROVED** [review]
✅ ✅

**Pull request merged**